### PR TITLE
Added reference to missing packages in README

### LINKED ISSUE #15621: README should refer to docs/README.rst for Dependencies
The [README](https://github.com/AnimeshSinha1309/SympyRepo/blob/master/README.rst) file in the repository suggests that we generate the docs using `make html` right away, but the docs do not compile. The file **docs/README.rst** includes what packages we need to install to compile the documentation.

This should be mentioned in the main _README_, that we go to _docs/README_ and check for the packages. Will make it easier to start off quick.

### PR DESCRIPTION
**Pull request by AnimeshSinha1309**
#### References to other Issues or PRs
Fixes #15621 - But does not change the apt-get install to pip because I am not sure that is a good idea.

#### Brief description of what is fixed or changed
Made minor updates to the `README.rst` and `docs/README.rst` helping new users install dependencies when building docs

#### Other comments
* To `README.rst`
  - In the build docs section, added reference to docs/README.rst if package requirements are not satisfied.
* To `docs/README.rst`
  - Reduplicated the segment on installing python-mpmath if build fails in the Ubuntu/Debian segment.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: doc/README.rst [hunk 1]
```diff
@@ -16,6 +16,10 @@ and to view it, do::
 
     firefox _build/html/index.html
 
+If you get **mpmath** error, install python3-mpmath package::
+
+    apt-get install python3-mpmath
```
Comment (asmeurer) [inline doc/README.rst]: I would just add this to the apt-get command above.

**Comment by AnimeshSinha1309** [comment]
@asmeurer I sure can add `python3-mpmath` or `python-mpmath` to the apt requirements in the install list. But I wanted to ask that should we not do this via **pip** instead of *apt-get*?

**Comment by asmeurer** [comment]
I don't know what the general recommendation is for Linux whether to use pip or install the apt package? Someone who uses it will need to say. I guess pip also works even if you aren't using the system Python, which can also be a plus.

**Comment by asmeurer** [comment]
Maybe the Python packages should be listed separately, noting you can install them with pip or with apt. Actually you also need sphinx, docutils, and matplotlib, see https://github.com/sympy/sympy/blob/master/.travis.yml#L251 (doctr is only required for the Travis build so ignore that).

**Pull request closed without merging**